integreat_cms/textlab_api/textlab_api_client.py

```diff
@@ -25,6 +27,7 @@
     "countInfinitiveConstructions": "infinitive-constructions",
     "countNominalStyle": "nominal-sentences",
     "countFutureTenseInSentence": "future-tense-sentences",
+    'dataTerms."1289".result': "abbreviations",
```

Comment:
I think this mapping could also be moved to utils, as it's not used here anymore